src/components/Domain/CopyableTextField.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {View} from 'react-native';
+import ActivityIndicator from '@components/ActivityIndicator';
+import CopyTextToClipboard from '@components/CopyTextToClipboard';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type CopyableTextFieldProps = {
+    /** Text to display and to copy */
+    value?: string;
+
+    /** Should an activity indicator be shown instead of the text and button */
+    isLoading?: boolean;
+};
+
+const ACTIVITY_INDICATOR_SIZE = 24;
```

Comment:
Why is this here and not centralized? this should be same const anywhere we use something similar